pip_audit/_dependency_source/resolvelib/pypi_provider.py

```diff
@@ -213,7 +213,10 @@ def get_project_from_index(
     response.raise_for_status()
     data = response.content
     doc = html5lib.parse(data, namespaceHTMLElements=False)
-    for i in doc.findall(".//a"):
+    links = doc.findall(".//a")
+    if not links:
+        raise PyPINotFoundError
```

Comment:
Tiny nit, but it might make sense to specialize the error slightly here (since PyPI *does* have a project registered for the name, but the project is just missing releases).

Reply:
Actually, thinking about this more, I don't think it makes sense to distinguish these error states. They're effectively same from the `pip` end user's perspective, so they should be the same from a `pip-audit` user's perspective.